packages/client/tests/functional/extensions/enabled/prisma/_schema.ts

```diff
@@ -51,16 +19,12 @@ export default testMatrix.setupSchema(({ provider }) => {
     firstName String
     lastName String
     posts Post[]
-    ${createModelToManyField(20)}
   }
 
   model Post {
     id ${idForProvider(provider)}
     user User @relation(fields: [userId], references: [id])
     userId ${foreignKeyForProvider(provider)}
   }
-
-  // the following models aren't used in the tests, but are here to test the performance of the extension
-  ${createModel(20, provider)}
```

Comment:
Not needed anymore, just makes things slower. This is tested in `prisma/packages/client/tests/e2e/issues/16600-tsc-crash-extensions` with 78 models and nested relations.